### PR TITLE
GitHub team config option

### PR DESCRIPTION
**Pull request by pablolee**
Let's use the Github Team members list instead of typing out every member

### CODE REVIEW

**Comment by createdbypete** [comment]
:+1:

**Comment by stephnacios** [comment]
Nice :+1:

**Comment by johnsyweb** [comment]
GitHub teams! Nice. I recommend sending this upstream!

**Comment by pablolee** [comment]
Good call. Shall do so.

**Comment by pablolee** [comment]
@johnsyweb err really n00b question but how do I do that? I tried creating a PR off of this branch against binaberry/seal master but ends up including a bunch of envato-specific stuff.

File: spec/seal_spec.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
     {
       'lion' => {
         'members' => [],
+        'github_team_ids' => [123],
```
Comment (johnsyweb) [inline spec/seal_spec.rb]: Can we do this by team _name_ rather than _ID_?

**Comment by johnsyweb** [comment]
Try creating a branch off binaberry/master and cherry-picking these two commits.

File: spec/seal_spec.rb (hunk 1 above)
Comment (pablolee) [inline spec/seal_spec.rb]: I wanted to do that but the Oktokit client asked for a team id which I'm not sure how to get (I had to go digging in the source code of our team for something that looked like an id)

**Pull request merged**